crates/daemon/src/monitor.rs

```diff
@@ -1075,6 +1086,20 @@ async fn batch_script_get_history(
     histories
 }
 
+static SYNC_DURATION_HISTOGRAM: conquer_once::Lazy<prometheus::Histogram> =
+    conquer_once::Lazy::new(|| {
+        prometheus::register_histogram!(
+            "monitor_sync_duration_seconds",
+            "The duration of one sync run of the monitor.",
+            vec![
+                10.0, 20.0, 30.0, 40.0, 50.0, 60.0, 70.0, 80.0, 90.0, 100.0, 110.0, 120.0, 130.0,
```

Comment:
Is 10 seconds the shortest you would expect?

Reply:
Fair questions - I think if we go under it would mean there are not many (long running) open positions. 
Given the current data we will likely not go under 10, but I'm happy to add a bucket for 5 and 2 as well.